### PR TITLE
Add version_description to ec2_launch_template

### LINKED ISSUE #1762: Add version_description to ec2_launch_template
### Summary

I found that `ec2_launch_template` does not expose `VersionDescription` from `create_launch_template_version`.

- [create_launch_template_version](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/ec2/client/create_launch_template_version.html)

### Issue Type

Feature Idea

### Component Name

ec2_launch_template

### Additional Information

```yaml (paste below)

```

### Code of Conduct

- [X] I agree to follow the Ansible Code of Conduct

### PR DESCRIPTION
**Pull request by squirrel532**
##### SUMMARY
Add `version_description` to ec2_launch_template module, which allows user update the `VersionDescription` of a launch template.

Fix #1762

##### ISSUE TYPE

- Feature Pull Request

##### COMPONENT NAME

ec2_launch_template

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by squirrel532** [comment]
@markuman I'm finished. Please review it.

File: plugins/modules/ec2_launch_template.py [hunk 1]
```diff
@@ -38,6 +38,11 @@
     - Which version should be the default when users spin up new instances based on this template? By default, the latest version will be made the default.
     type: str
     default: latest
+  version_description:
+    description:
+    - The description of a launch template version.
+    default: ""
+    type: str
```
Comment (markuman) [inline plugins/modules/ec2_launch_template.py:45]:
```suggestion
  version_description:
    version_added: 5.5.0
    description:
    - The description of a launch template version.
    default: ""
    type: str
```

File: changelogs/fragments/1763-ec2-launch-template-add-version-description.yml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - ec2_launch_template - Add argument to config `VersionDescription` of a EC2 launch template.
```
Comment (markuman) [inline changelogs/fragments/1763-ec2-launch-template-add-version-description.yml:2]:
```suggestion
  - ec2_launch_template - Add argument to config `VersionDescription` of a EC2 launch template (https://github.com/ansible-collections/community.aws/pull/1763).
```

**Comment by alinabuzachis** [comment]
recheck

**Comment by markuman** [comment]
@squirrel532 we need some darker formations.  
can you also apply them please? https://github.com/ansible-collections/community.aws/actions/runs/4594116048/jobs/8112755969?pr=1763

**Review by markuman — APPROVED** [review]

**Pull request merged**